marina-latkova/README.md

```diff
@@ -0,0 +1,92 @@
+# **Flo Health - Period Tracker User Documentation**
+<img src= "img/flo.png" alt= “” width="1000" height="500">
+
+### Welcome to Flo Health - Period Tracker, your personal mobile app for tracking your menstrual cycle, fertility, and overall reproductive health. This user documentation will guide you through the various features and functionalities of the Flo Health - Period Tracker mobile app. Let's get started!
+<br />
+
+<img src="img/floo.png" alt= “” width="950" height="190">
+
+<br />
+<br />
+
+ > ## **Table of Contents**
+ <br />
+
+1. Installation and Account Setup
+2. Entering Menstrual Cycle Data
+3. Tracking Menstrual Cycle
+4. Recording Symptoms and Moods
+5. Health Insights and Articles
+6. Settings and Customization
+7.  Troubleshooting  
+<br />
+
+> ## **1. Installation and Account Setup:**
+<br />
+
+* Download and install Flo Health - Period Tracker from your device's app store (available for both iOS and Android).
+* Open the app and sign up for a new account or log in using your existing account credentials.
+* Complete the initial setup process, including entering basic information such as your date of birth and cycle length.
+
+<br />
+
+> ## **2. Entering menstrual cycle data:**
+<br />
+
+* To accurately track your menstrual cycle, enter the start and end dates of your periods.
+* Tap the "+" icon on the calendar and select the start date of your period.
+* If your period lasts multiple days, adjust the duration accordingly.
+* Save the information, and your period will be recorded on the calendar.
+
+<br />
+
+> ## **3. Tracking Menstrual Cycle:**
+<br />
+
+* The calendar view displays your menstrual cycle, fertility window, and predicted ovulation dates.
+* The app calculates the average length of your cycle based on your entered data and predicts future periods.
+* Tap on a specific date to view more detailed information about that day, including symptoms and moods.
+
+<br />
+
+> ## **4. Recording Symptoms and Moods:**
+<br />
+
+* Flo Health - Period Tracker allows you to record and track various symptoms and moods throughout your cycle.
+* Tap the "+" icon on the calendar or the "Add" button on the symptoms page to record a new symptom or mood.
```

Comment:
When highlighting this or that UI element in the text, please make it bold instead of adding "".
(See https://learn.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions, the main recommendation here is to use bold).